docs/user_guide/Client_Management.rst

```diff
@@ -52,7 +52,7 @@ can use an existing DNS infrastructure.
      * ``named/var/named/forward/your.domain.db``
      * ``named/var/named/reverse/0.0.10.db``
 
-   * Review ``named/etc/named.conf`` and update the following:
+   * Make the following chages to ``named/etc/named.conf``:
```

Comment:
Review makes sense because there may be other settings that need to be incorporated/changed for the site.